bigbluebutton-html5/imports/ui/components/tabs/components/TabContent/TabContentContainer.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import Service from '../../service';
+import PresentationService from '/imports/ui/components/presentation/presentation-uploader/service';
+import TabContent from './TabContent';
+
+const TabContentContainer = (props) => {
+  const options = [];
+
+  const allOptionPages = [];
+
+  let Pages = null;
+
+  const {
+    presentations, fileType, onSelectoption, selectedOption,
+  } = props;
+
+  presentations.forEach((element) => {
+    options.push({
+      value: element.id,
+      label: element.filename,
+    });
+
+    allOptionPages[element.id] = element.pages.sort((a, b) => a.num - b.num);
+  });
+
+  Pages = allOptionPages[selectedOption];
+
+  const NoOptionMessage = (fileType === 'PDF') ? 'No Documents' : 'No Presentations';
+
+  const SlideType = (fileType === 'PDF') ? 'Page' : 'Slide';
```

Comment:
move this label to constants

```
{
  fileType:..,
  icon:..,
  emptyMessage:..,
  slideLabel:..,
}
```

and use over here

Reply:
fixed